pkg/compiler/codegen.go

```diff
@@ -1222,6 +1222,8 @@ func (c *codegen) convertBuiltin(expr *ast.CallExpr) {
 		c.emitConvert(typ)
 	case "SHA256":
 		emit.Syscall(c.prog.BinWriter, "Neo.Crypto.SHA256")
+	case "RIPEMD160":
```

Comment:
Not sure why these hash syscalls are builtins. Can we move them to `compiler/syscall.go`?

Reply:
`SHA256` was a builtin back when we have a separate opcode for it. I guess now is the time to treat is like other syscall.